### PR TITLE
php-cs-fixer fixes

### PR DESCRIPTION
**Pull request by mbadolato**
Ran php-cs-fixer with the .php_cs file

### CODE REVIEW

File: examples/event-sourced-child-entity/Parts.php [hunk 1]
```diff
@@ -141,7 +141,7 @@ class PartManufacturerWasRenamedEvent
 
     public function __construct($partId, $manufacturerName)
     {
-        $this->partId = $partId;
+        $this->partId           = $partId;
```
Comment (cordoval) [inline examples/event-sourced-child-entity/Parts.php]: no please

Comment (mbadolato) [inline examples/event-sourced-child-entity/Parts.php]: That's part of the php_cs definition that Broadway uses. 

Some of us prefer having them lined up, some don't. But the broadway team chose to put that in their coding standards file. Personal preference is irrelevant.

Comment (cordoval) [inline examples/event-sourced-child-entity/Parts.php]: :baby: yes i remember now :+1: it is ok!

**Comment by wjzijderveld** [comment]
Already noticed some of these at time of the PR, but didn't want to comment it at the time.
I'm not sure if we should be more strict during PR review.

I would be fine if we keep it as clean as possible, but don't comment everything on every PR. Just running PHPCS once in a while might be sufficient.

 Once we have a stable 1.0 that would probably change.

Thanks @mbadolato

**Pull request merged**